tests/monitoring_integration_test.py

```diff
@@ -77,6 +78,31 @@ def getZone():
 
     return zone
 
+_services_short = [
```

Comment:
I would combine it into single array of tuples - what happens if you add a value in one array and forget the second?

Reply:
Good point, done